rust/automerge/Cargo.toml

```diff
@@ -15,8 +15,8 @@ utf8-indexing = []
 utf16-indexing = []
 
 [dependencies]
-#hexane = { path = "../hexane" }
-hexane = "0.1.1"
+hexane = { path = "../hexane" }
```

Comment:
You can put the version in here as well, it'll just choose the local version if available, then fall back to crates.io.